feature-libs/asm/root/interceptors/user-id-http-header.interceptor.spec.ts

```diff
@@ -0,0 +1,226 @@
+import {
+  HttpClient,
+  HttpContext,
+  HTTP_INTERCEPTORS,
+} from '@angular/common/http';
+import {
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { provideConfig } from '@spartacus/core';
+import { Observable, of } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+import { OCC_USER_ID_CURRENT } from '../../../../projects/core/src/occ/utils/occ-constants';
+import { UserIdService } from '../../../../projects/core/src/auth/user-auth/facade';
+import {
+  OCC_HTTP_TOKEN,
+  OCC_USER_ID_CONSTANTS,
+} from '../../../../projects/core/src/occ/utils';
```

Comment:
Are these imports appropriate?

Reply:
Ah, no. I suppose these are ignored for compilation because they're in unit tests, but they ought to be fixed. Ty.